sdks/java/extensions/schemaio-expansion-service/build.gradle

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id 'org.apache.beam.module'
+}
+
+applyJavaNature(
+        ignoreRawtypeErrors: true,
+        automaticModuleName: 'org.apache.beam.sdk.extensions.schemaio.expansion',
+        validateShadowJar: false,
+        shadowClosure: {
+            manifest {
+                attributes "Main-Class": "org.apache.beam.sdk.expansion.service.ExpansionService"
+            }
+        }
+)
+
+dependencies {
+    compile project(path: ":sdks:java:expansion-service")
+    compile project(":sdks:java:io:jdbc")
+    compile library.java.postgres
+    testCompile library.java.junit
+    testCompile library.java.powermock_mockito
+    testCompile library.java.mockito_core
+    testCompile "org.checkerframework:checker-qual:3.5.0"
+}
+
+task runExpansionService (type: JavaExec) {
+    main = "org.apache.beam.sdk.expansion.service.ExpansionService"
+    classpath = sourceSets.main.runtimeClasspath
+    args = [project.findProperty(":constructionService.port") ?: "8097"]
+}
```

Comment:
I don't think we need this task, Python is responsible for running the expansion service from the compiled jar using subprocess.

Reply:
Done.